src/control-flow-basics/loops.md

```diff
@@ -8,51 +8,15 @@ There are three looping keywords in Rust: `while`, `loop`, and `for`:
 
 ## `while`
```

Comment:
To avoid students seeing an empty "table of contents" slide, let's include the first section (`while`) here in `src/control-flow-basics/loops.md`.

Reply:
ok